.github/actions/occt-install-dep/action.yml

```diff
@@ -62,22 +63,6 @@ runs:
       shell: bash
       run: cmake --build . --parallel 2 --target install --config Release
 
-    - name: Patch OCCT install macos
```

Comment:
This has been patched upstream? Nice!

Reply:
no, but the patch does not seem needed when building more modules